doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc

```diff
@@ -1,24 +1,95 @@
+image:ssui-services-menu.png[]
 
-image:SSUI_Service_Details1.png[]
+There is now the option to perform most of these tasks in bulk (except editing service details and power operations). In order to do so, select the services and then click the designated button for the required task on the upper-left corner of the screen. 
 
-Moreover, you can now add custom buttons and custom button groups that can have dialogs associated with them. For example:
+Additionally, click on a service to view more details, including information on associated virtual machines and snapshots, and its relationships data.
 
-image:SSUI_Custom_Button1.png[]
+[[html5-console]]
+=== Accessing the HTML5 Console
+
+Follow this procedure to get HTML5 console access to the virtual machine. The console is the remote control system that enables you to work and interact directly with your virtual machines. 
+
+. From the *Services* screen, click on the required service.
+. In *Resources*, click the image:ssui-access-button.png[Access Button] button next to the virtual machine and select *Open HTML5 Console*.
+image:ssui-access-console.png[]
+
+The HTML5 console will open in a new window of your web browser. Note that you may have to enable the pop-up settings in your browser to access the console.
+
+[[schedule-retirement]]
+=== Setting a Retirement Date for a Service
+
+Follow this procedure to schedule retirement for a service at a later date. For retiring a service immediately, see xref:retiring-service[].
+
+. From the *Services* screen, click on the required service you want to schedule retirement for.
+. Click the image:ssui-retirement-button.png[Service Retire Button] button located on the upper-right corner of the screen, then select *Set Retirement Dates*. The *Schedule Service Retirement* window will appear.
+image:ssui-schedule-service-retirement.png[]
+. Set the *Retirement Date*.
+. Select the *Retirement Warning* from the list.
+. Click *Save*.
+
+[[retiring-service]]
+=== Retiring a Service
+
+Follow this procedure to retire a service immediately. For retiring a service at a later date, see xref:schedule-retirement[].
+
+. From the *Services* screen, click on the service you want to retire now.
+. Click the image:ssui-retirement-button.png[Service Retire Button] button located on the upper-right corner of the screen, then select *Retire*. The *Retire Service Now* window will appear to confirm your action.
+image:ssui-retire-service-now.png[]
+. Click *Yes, Retire Service Now*.
+
+=== Setting Ownership of a Service
+
+Follow this procedure to set ownership of a service. 
+
+. From the *Services* screen, click on more options next to the service you want set ownership for, then click *Set Ownership*.
+. Alternatively, click on the service. Click the image:ssui-service-settings-button.png[] button located on the upper-right corner of the screen, then select *Set Ownership*.
```

Comment:
[EDITED] I combined these steps - please feel free to review and restructure if needed.